<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainmenu</class>
 <widget class="QDialog" name="Mainmenu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btNum1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBLDBT">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNum0">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBRDBT">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btStop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btCancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDBT">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btAccept">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelOption1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option2</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option4</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option3</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option6</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option7</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option8</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOption5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Option5</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>input</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrompt">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prompt</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btOption4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOption6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBankSimul_R9">
    <property name="title">
     <string>BankSimul_R9</string>
    </property>
   </widget>
   <addaction name="menuBankSimul_R9"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
